src/applications/personalization/profile/tests/e2e/contact-information/confirm-email-address-no-email.cypress.spec.js

```diff
@@ -0,0 +1,116 @@
+import { PROFILE_PATHS } from '@@profile/constants';
+import mockUser from '@@profile/tests/fixtures/users/user-36.json';
+import {
+  mockFeatureToggles,
+  mockGETEndpoints,
+} from '@@profile/tests/e2e/helpers';
+
+const MHV_EMAIL_CONFIRMATION_DISMISSED_COOKIE =
+  'MHV_EMAIL_CONFIRMATION_DISMISSED';
+
+const setup = () => {
+  mockFeatureToggles(() => ({
+    data: {
+      type: 'feature_toggles',
+      features: [
+        {
+          name: 'mhvEmailConfirmation',
+          value: true,
+        },
+      ],
+    },
+  }));
+
+  const mockUserNoEmail = { ...mockUser };
+  mockUserNoEmail.data.attributes.vet360ContactInformation.email.emailAddress = null;
+  cy.login(mockUserNoEmail);
+
+  mockGETEndpoints([
+    'v0/mhv_account',
+    'v0/profile/full_name',
+    'v0/profile/status',
+    'v0/profile/personal_information',
+    'v0/profile/service_history',
+    'v0/profile/direct_deposits',
+  ]);
+
+  cy.intercept('GET', '/v0/user?*', {
+    statusCode: 200,
+    body: mockUserNoEmail,
+  }).as('finalUserRequest');
+
+  cy.visit(PROFILE_PATHS.CONTACT_INFORMATION);
+  cy.wait('@mockUser');
+};
+
+const clickAddEmailAddress = () => {
+  cy.get('va-alert[data-testid="profile-alert--add-contact-email"]')
+    .find('va-button-pair')
+    .shadow()
+    .find('va-button')
+    .first()
+    .shadow()
+    .find('button')
+    .click();
+};
+
+const clickSkipAddingEmailAddress = () => {
+  cy.get('va-alert[data-testid="profile-alert--add-contact-email"]')
+    .find('va-button-pair')
+    .shadow()
+    .within(() => {
+      cy.get('va-button[secondary]')
+        .shadow()
+        .find('button')
+        .click();
+    });
+};
+
+describe('MHV Email Confirmation Alert - Add Email', () => {
+  beforeEach(() => {
+    setup();
+    // Clear the MHV email confirmation alert cookie to ensure clean test state
+    cy.clearCookie(MHV_EMAIL_CONFIRMATION_DISMISSED_COOKIE);
+  });
+
+  it('should not show the error alert when the feature flag is disabled', () => {
+    mockFeatureToggles(() => ({
+      data: {
+        type: 'feature_toggles',
+        features: [
+          {
+            name: 'mhvEmailConfirmation',
+            value: false,
+          },
+        ],
+      },
+    }));
+
+    cy.findByTestId('profile-alert--confirm-contact-email').should('not.exist');
+
+    cy.injectAxeThenAxeCheck();
+  });
+
+  it('should focus the email address field when "Add a contact email" is clicked', () => {
+    clickAddEmailAddress();
+
+    cy.get('va-text-input[name="root_emailAddress"]')
```

Comment:
Same as the other file. Need a cy.wait, another cy.find, or change the gets above to finds.